python/paddle/fluid/layers/nn.py

```diff
@@ -1369,6 +1370,171 @@ def _get_default_param_initializer():
     return helper.append_activation(pre_act)
 
 
+def conv3d(input,
+           num_filters,
+           filter_size,
+           stride=1,
+           padding=0,
+           dilation=1,
+           groups=None,
+           param_attr=None,
+           bias_attr=None,
+           use_cudnn=True,
+           use_mkldnn=False,
+           act=None,
+           name=None):
+    """
+    **Convlution3D Layer**
```

Comment:
Can use `@templatedoc` to avoid copying comments, see yu laoshi's PR: https://github.com/PaddlePaddle/Paddle/pull/11308

Reply:
@typhoonzero  Sound great!
But `Convlution Layer` and `Convlution Op` are not completely equal. 
The equation of `Convlution Layer` is $Out = \sigma (W \\ast X + b)$. But `Convlution Op`'s is  $Out = W \\ast X$. So their comment should not be the same.

Reply:
I see.